chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -83,6 +83,10 @@
   # Stop remaining pacemaker resources
   bash "stop pacemaker resources" do
     code <<-EOF
+      crm stop postgresql
+      # HA: Deletes postgres data directory after stopping postgresql service
+      # but before stopping fs-postgresql.
+      rm -rf /var/lib/pgsql/data
```

Comment:
> But the other part gets anyway executed?

That's true, but in the HA case it's too late then, because the fs-postgresql resource is stopped and /var/lib/pgsql/data/ is umounted already.

That being said I guess the code could be re-arranged to first stop postgresql (using crm in HA and init-script in non-HA) then delete/move the directory and after that stop the rest of the services. I am not quite sure if that will make the code easier though.